msteams-platform/m365-apps/prerequisites.md

```diff
@@ -71,7 +71,7 @@ For more information on Microsoft 365 release options, see [set up the Standard
 
 ### Desktop
 
-You can preview Teams apps running in Outlook on Windows desktop by using a recent *Current Channel (Preview)* build. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 Current Channel (Preview) build.
+You can preview Teams apps running in Outlook on Windows desktop by using a recent *Current Channel* build. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 Current Channel build.
```

Comment:
Let's update this to whatever we land on for this PR: https://github.com/MicrosoftDocs/msteams-docs/pull/9491